packages/playground/src/utils/getMetricsUrl.ts

```diff
@@ -24,15 +24,19 @@ export class GrafanaStatistics implements IGrafanaArgs {
   }
 
   async setAccountID() {
-    await this.grid.twins.get({ id: this.deployment[0].nodeId }).then(res => {
-      this.accountID = res.accountId;
-    });
+    if (this.deployment.length) {
+      await this.grid.twins.get({ id: this.deployment[0].nodeId }).then(res => {
+        this.accountID = res.accountId;
+      });
+    }
   }
 
   async setfarmID() {
```

Comment:
i noticed it's not in the correct camel case, should be `setFarmID` instead of `setfarmID`

Reply:
updated